crates/relayer-types/src/clients/ics07_axon/header.rs

```diff
@@ -2,38 +2,26 @@ use crate::core::ics02_client::{self, error::Error as Ics02Error};
 use crate::prelude::*;
 use crate::timestamp::Timestamp;
 use crate::Height;
-use axon_tools::types::AxonHeader;
 use ibc_proto::google::protobuf::Any;
 use ibc_proto::protobuf::{Error as ProtoError, Protobuf};
 use serde::{Deserialize, Serialize};
 
 pub const AXON_HEADER_TYPE_URL: &str = "/axon.v1.header";
 
 #[derive(Clone, PartialEq, Eq, Deserialize, Serialize, Debug)]
-pub struct Header {
-    pub axon_header: AxonHeader,
-}
-
-impl From<AxonHeader> for Header {
-    fn from(header: AxonHeader) -> Self {
-        Self {
-            axon_header: header,
-        }
-    }
-}
+pub struct Header {}
 
 impl ics02_client::header::Header for Header {
     fn client_type(&self) -> ics02_client::client_type::ClientType {
         ics02_client::client_type::ClientType::Axon
     }
 
     fn height(&self) -> Height {
-        Height::new(u64::MAX, self.axon_header.number).expect("bad axon block number")
+        Height::new(1u64, u64::MAX).expect("bad axon block number")
```

Comment:
We should leave comments on the two functions.

Reply:
nice suggestion, mark it